03 Misc/08 DefinePlugin advanced/src/averageService.js

```diff
@@ -0,0 +1,9 @@
+export function getAvg(scores) {
```

Comment:
Should we remove averageService and display total components and only focus on the new student list component? (just to remove noise from the sample)